samples/snippets/csharp/VS_Snippets_CLR_System/system.Linq.Enumerable/CS/enumerable.cs

```diff
@@ -341,7 +341,7 @@ static void AverageEx3()
             // <Snippet16>
             string[] numbers = { "10007", "37", "299846234235" };
 
-            double average = numbers.Average(num => Convert.ToInt64(num));
+            double average = numbers.Average(num => long.Parse(num));

```

Comment:
I could've made it a method group, but like previous time, this one is probably more understandable for newcomers

Reply:
I like the current change.